app/jinja_filters.py

```diff
@@ -648,6 +648,31 @@ def map_summary_item_config_processor() -> dict[str, Callable]:
     return {"map_summary_item_config": map_summary_item_config}
 
 
+@blueprint.app_context_processor
+def map_list_config_processor() -> dict[str, Callable]:
+    return {"map_list_config": map_list_config}
+
+
+@blueprint.app_template_filter()
+def map_list_config(list_values: list[dict]) -> list[dict]:
+    items_list: list[dict] = []
+
+    for index, value in enumerate(list_values, 1):
+        item: dict = {"text": value["item_title"]}
+
+        if value["is_complete"]:
+            item["iconType"] = "check"
+
+        item["attributes"] = {
+            "data-qa": f"list-item-{index}-label",
+            "data-list-item-id": value["list_item_id"],
```

Comment:
Are we doing anything with `data-list-item-id`? Is it needed?

Reply:
Used in tests (integration).

Reply:
But not for List Collector Content pages?

Reply:
Don't think there is an integration test that calls this new method, so probably a sign that one needs adding

Reply:
Ok, seems like we have a gap in our testing, probably because map_list_config is referenced in the jinja processor and used as a blueprint we didn't get the lower coverage (it goes undetected).

Reply:
Still not sure `data-list-item-id` is needed for this as looks like some other list collector features use it as part of their integration tests, but we're not really doing anything with it here. I'd just remove it.